lib/core/src/lib/comments/comment-list/comment-list.component.scss

```diff
@@ -1,68 +1,70 @@
-.adf {
-    &-comment-img-container {
-        width: 40px;
-        height: 100%;
-        display: flex;
-        align-self: flex-start;
-        padding-top: 18px;
-    }
+.adf-comment-list {
+    .adf {
+        &-comment-img-container {
+            width: 40px;
+            height: 100%;
+            display: flex;
+            align-self: flex-start;
+            padding-top: 18px;
+        }
 
-    &-comment-list-item {
-        white-space: initial;
-        display: table-row-group;
-        padding-top: 12px;
-        overflow: hidden;
-        height: 100% !important;
-        background-position: center;
-    }
+        &-comment-list-item.mat-list-item {
+            white-space: initial;
+            display: table-row-group;
+            padding-top: 12px;
+            overflow: hidden;
+            height: 100%;
+            background-position: center;
+        }
 
-    &-comment-user-icon {
-        padding: 10px 5px;
-        width: 30px;
-        background-color: var(--theme-primary-color);
-        color: var(--theme-primary-color-default-contrast);
-        border-radius: 50%;
-        font-size: var(--theme-subheading-2-font-size);
-        text-align: center;
-        height: 20px;
-        background-size: cover;
-    }
+        &-comment-user-icon {
+            padding: 10px 5px;
+            width: 30px;
+            background-color: var(--theme-primary-color);
+            color: var(--theme-primary-color-default-contrast);
+            border-radius: 50%;
+            font-size: var(--theme-subheading-2-font-size);
+            text-align: center;
+            height: 20px;
+            background-size: cover;
+        }
 
-    &-comment-user-name {
-        width: 100%;
-        padding: 2px 10px;
-        font-weight: 600;
-        font-size: var(--theme-body-1-font-size);
-    }
+        &-comment-user-name {
+            width: 100%;
+            padding: 2px 10px;
+            font-weight: 600;
+            font-size: var(--theme-body-1-font-size);
+        }
 
-    &-comment-message {
-        width: 100%;
-        padding: 2px 10px;
-        white-space: pre-line !important;
-        font-size: var(--theme-body-1-font-size);
-        letter-spacing: -0.2px;
-        line-height: 1.43;
-        color: var(--adf-theme-foreground-text-color);
-    }
+        &-comment-message.mat-line {
```

Comment:
@jacekpluta I can see still usage of mat class here. Could you please replace it with something different?